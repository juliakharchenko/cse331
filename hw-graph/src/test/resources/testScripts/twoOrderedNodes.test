# Tests that nodes are printed in alphabetical order

# Create a graph
CreateGraph multiNodeGraph

# Add nodes
AddNode multiNodeGraph b
AddNode multiNodeGraph a

# Print the nodes in the graph in alphabetical order and the outgoing edges from a and b
ListNodes multiNodeGraph
ListChildren multiNodeGraph a
ListChildren multiNodeGraph b