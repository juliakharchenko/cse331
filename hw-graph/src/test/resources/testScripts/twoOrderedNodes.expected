# Tests that nodes are printed in alphabetical order

# Create a graph
created graph multiNodeGraph

# Add nodes
added node b to multiNodeGraph
added node a to multiNodeGraph

# Print the nodes in the graph in alphabetical order and the outgoing edges from a and b
multiNodeGraph contains: a b
the children of a in multiNodeGraph are:
the children of b in multiNodeGraph are: